tedana/workflows/t2smap.py

```diff
@@ -248,7 +259,11 @@ def _main(argv=None):
     else:
         logging.basicConfig(level=logging.INFO)
 
-    t2smap_workflow(**vars(options))
+    kwargs = vars(options)
+    n_threads = kwargs.pop('n_threads')
+    n_threads = None if n_threads == -1 else n_threads
+    with threadpool_limits(limits=n_threads, user_api=None):
+        t2smap_workflow(**vars(options))
```

Comment:
```suggestion
        t2smap_workflow(**kwargs)
```

I believe you want to provide `kwargs` here since you removed `n_threads` (which is not a parameter of `t2smap_workflow`). But I might be mistaken!

Reply:
Thank you for catching this!